src/cart/controllers/cart.controller.ts

```diff
@@ -23,34 +19,64 @@ export class CartController {
 
     @Get()
     @ApiOperation({summary: 'Gets all carts'})
+    @ApiResponse({status: 201, description: 'Listing all Carts'})
+    @ApiForbiddenResponse({ status: 403, description: 'Forbidden.'})
     getAll(): Promise<Cart[]>{
         return this.cartService.findAll();
     }
 
     @Get(':id')
+    @ApiParam({
+        name: "id",
+        type: "integer",
+        required: true
+    })
     @ApiOperation({summary: 'Gets one cart by Id'})
+    @ApiResponse({status: 201, description: 'Cart succesfully found'})
+    @ApiForbiddenResponse({ status: 403, description: 'Forbidden.'})
+    @ApiResponse({status: 404, description: 'No Cart was found that matches that id'})
```

Comment:
Sebas, una consulta, es correcto que esté repetido el decorator @ApiResponse? 🤔 lo que usé en payment methods es un @ApiNotFoundResponse para el caso de `'No Cart was found that matches that id'`

Reply:
ApiResponse es el generico que te permite crear cualquier response que quieras, esta bien usar el @ApiNotFoundResponse me parece que ese decorador le pone el 404 de status por default